staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -235,12 +235,12 @@ const (
 	createTagFactor       = 2.0
 	createTagSteps        = 9
 
-	// encryptedCheck* is configuration of poll for created volume to check
+	// createEBSCheck* is configuration of poll for created volume to check
 	// it has not been silently removed by AWS.
 	// On a random AWS account (shared among several developers) it took 4s on
 	// average.
-	encryptedCheckInterval = 1 * time.Second
-	encryptedCheckTimeout  = 30 * time.Second
+	createEBSCheckInterval = 5 * time.Second
```

Comment:
Let's keep it the same as EBS CSI driver: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/pkg/cloud/cloud.go#L742